ViV-Wallet-api/src/main/java/com/invivoo/vivwallet/api/domain/action/Action.java

```diff
@@ -1,5 +1,6 @@
 package com.invivoo.vivwallet.api.domain.action;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
why is there so much java code in this PR?

Reply:
Just rebased